### PR TITLE
adding truss for whisper streaming

### PR DESCRIPTION
**Pull request by htrivedi99**

### CODE REVIEW

File: whisper/whisper-streaming/config.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  image: baseten/truss-server-base:3.10-gpu-v0.4.9
+  python_executable_path: /usr/bin/python3
+environment_variables: {}
+external_package_dirs: []
+model_metadata:
+  whisper_model: medium
+model_name: Whisper Streaming
+python_version: py310
+requirements:
+  - torch==2.1.0
```
Comment (bolasim) [inline whisper/whisper-streaming/config.yaml:11]: let's do this in a requirements file if possible.

File: whisper/whisper-streaming/config.yaml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  - faster-whisper==0.10.0
+  - librosa==0.10.1
+  - opus-fast-mosestokenizer==0.0.8.5
+resources:
+  accelerator: T4
+  cpu: "2"
```
Comment (bolasim) [inline whisper/whisper-streaming/config.yaml:17]: you can drop cpu and memory

**Review by bolasim — APPROVED** [review]
is the stuff in packages just a `git clone` of their code?

**Comment by htrivedi99** [comment]
> is the stuff in packages just a `git clone` of their code?

yeah, pretty much

**Pull request merged**